web/bitwarden/bitwarden.go

```diff
@@ -292,6 +286,20 @@ func getInitialCredentials(c echo.Context) error {
 	}
 	key := setting.Key
 
+	if _, err := setting.OrganizationKey(); err == settings.ErrMissingOrgKey {
+		// The organization key should exist at this moment as it is created at the
+		// instance creation or at the login-hashed migration.
+		log.Warningf("Organization key does not exist")
```

Comment:
Is it useful to repeat the same warning that we have at the prelogin step?

Reply:
I removed it with the new commit (the warning was in `SetKeyPair`)